WcaOnRails/app/controllers/api/v0/user_roles_controller.rb

```diff
@@ -8,6 +8,40 @@ class Api::V0::UserRolesController < Api::V0::ApiController
     end
   end
 
+  private def status_sort_rank(status)
```

Comment:
What about `['leader', 'senior_member', 'member'].find_index(status) || 4`?

You could take this one step further and move the sorted array into a `STATUS_SORTING_ORDER` constant like so:
```ruby
STATUS_SORTING_ORDER.find_index(status) || STATUS_SORTING_ORDER.length # this ensures that the return value "grows" properly when the sorting order is adjusted later on
```